### ISSUE TITLE
HH Calibration for multiple layers with same material 

### ISSUE THREAD

**Issue by domniki** [jsh9/PySeismoSoil#3]
G/Gmax, damping and strength arrays are required to match the number of layers, not the number of materials. In case of many layers with same materials (thin layers with different Vs but same MRD curves and strength) one needs to write the curves and strength files with the same material properties many times.

**Comment by jsh9** [comment]
Yes, you are right.  I found this annoying too.  But this would require some redesign of the input file format.  (For example, we need some mapping to indicate which material maps to which strength, and which material maps to which layer.)

If one of your students can implement this redesign, I'd happy to offer my suggestions.
